### DISCUSSION TITLE
GA release of Janusgraph to compatibility with ElasticSearch 8.6.x version

### DISCUSSION THREAD

**Discussion by himir11285** [JanusGraph/janusgraph#3690] [Q&A] [answered]
Currently pre release version v1.0.0-rc2 compatible with Elasticsearch version 8.6.0 version. 
when Janusgraph planning to GA release to compatibility with Elasticsearch version  8.6.x ?

**Comment by porunov** [comment, marked as answer]
JanusGraph 1.1.0 supports ElasticSearch 8. 
Unreleased version JanusGraph 1.2.0 supports ElasticSearch 9.
For people who use commit releases, the version from which JanusGraph started to support ElasticSearch 9 is:  `1.2.0-20250812-182757.f621a1c`.
